src/constants/WorkflowTemplates.ts

```diff
@@ -434,6 +434,235 @@ const createHyperliquidBTCSpotNPerpsThresholdNotificationWorkflow = () => {
+
+        return workflow;
+    }
+}
+
+const createTwitterAiNotificationTemplate = (username: {display: string, tag: string}, prompt: string) => {
+  return {
+    'name': `Get notified when ${username.display} tweets ${prompt}`,
+    'description': `Get notified when ${username.display} tweets ${prompt}`,
+    'tags': [WORKFLOW_TEMPLATES_TAGS.SOCIALS, WORKFLOW_TEMPLATES_TAGS.NOTIFICATIONS],
+    'thumbnail': 'https://otomato-sdk-images.s3.eu-west-1.amazonaws.com/templates/dailyYieldUpdates.jpg',
+    'image': [
+        TRIGGERS.SOCIALS.X.image,
+        ACTIONS.AI.AI.image,
+        ACTIONS.CORE.CONDITION.image,
+        ACTIONS.NOTIFICATIONS.TELEGRAM.image
+    ],
+    'blockIDs': [
+        TRIGGERS.SOCIALS.X.X_POST_TRIGGER.blockId,
+        ACTIONS.AI.AI.AI.blockId,
+        ACTIONS.CORE.CONDITION.IF.blockId,
+        ACTIONS.NOTIFICATIONS.TELEGRAM.SEND_MESSAGE.blockId
+    ],
+    createWorkflow: createTwitterAiNotificationWorkflow(username, prompt)
+  }
+}
+
+const twitterAiTemplates = [
+  createTwitterAiNotificationTemplate(
+    {display: 'Hyperliquid', tag: 'HyperliquidX'},
+    'announces a new listing'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'Hyperliquid', tag: 'HyperliquidX'},
+    'mentions season 2, an airdrop, a liquidity mining campaign, or any trading incentives'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'lookonchain', tag: 'lookonchain'},
+    'mentions that AAVE has been hacked'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'MegaETH', tag: 'megaeth_labs'},
+    'announces mainnet or TGE'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'Somnia', tag: 'Somnia_Network'},
+    'announces mainnet or TGE'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'Monad', tag: 'monad'},
+    'announces mainnet or TGE'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'Binance', tag: 'binance'},
+    'announces a new token listing'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'VitalikButerin', tag: 'VitalikButerin'},
+    'is bullish about Ethereum'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'Aave', tag: 'aave'}, 
+    'announces a major partnership'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'MorphoLabs', tag: 'MorphoLabs'},
+    'announces a major partnership'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'saylor', tag: 'saylor'},
+    'announces that Microstrategy acquired more bitcoin'
+  ),
+  createTwitterAiNotificationTemplate(
+    {display: 'phtevenstrong', tag: 'phtevenstrong'},
+    'is about yield opportunities'
+  ),
+];
+
+const createBuyBitcoinOnPeterSchiffBearishWorkflow = (): Workflow => {
+    const trigger = new Trigger(TRIGGERS.SOCIALS.X.X_POST_TRIGGER);
+    trigger.setParams('username', 'PeterSchiff');
+    trigger.setPosition(400, 120);
+
+    const aiAction = new Action(ACTIONS.AI.AI.AI);
+    aiAction.setParams('prompt', `this tweet mention bearish`);
```

Comment:
```suggestion
    aiAction.setParams('prompt', `Analyze the sentiment of this tweet by Peter Schiff. Return true if the tweet is negative, bearish, or critical about Bitcoin in any way. Otherwise, return false.`);
```